pkg/config/node.go

```diff
@@ -23,6 +23,11 @@ import (
 	"github.com/openshift/installer/pkg/types"
 )
 
+const (
```

Comment:
Nit: This section could be combined with the const below.

Reply:
done